andela/ah-frontend-targaryen
# #160609517 view profiles

### What does this PR do?

It enables users to view their profiles

### Description of tasks to be completed?

Currently, a user cannot view his/her profile
This PR adds functionality to allow users to view their profiles

### How should this be manually tested?

- Run `npm install` to install all dependencies
- Run `npm start`
- Log a user into Auhtors Haven
- Click `profiles` on the navigation bar

You should be able to view your profile.

### What are the relevant pivotal tracker stories?

[#160609517](https://www.pivotaltracker.com/story/show/160609517)

### Any background context you want to add?

N/A

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`


  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
